README.md

```diff
@@ -1,3 +1,13 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+instruction for connecting to the databases:
+
+Create two .env files : 
+
+<.env.test> 
+<.env.development> 
+
+
+Into each, add PGDATABASE=, with the correct database name for that environment (these names can be found in the setup.sql)
+
```

Comment:
This instruction can be more explicit. We can't assume the person using this repo knows what to look for in the setup.sql. I'd say exactly what to put in each file